### DISCUSSION TITLE
对于Arm64和x86的支持

### DISCUSSION THREAD

**Discussion by iMallpa** [localsend/localsend#580] [L-简体中文]
如题，我这里有一台运行着ARM架构处理器的电脑，我的朋友在得知我需要一个跨设备互传软件时极力推荐我尝试他，但是当我下载.msix文件运行时，Windows显示不支持我的设备，我很沮丧
我真心喜欢Local send，并且希望它可以适配更多的平台，为更多平台的用户提供优质的服务
感谢

**Comment by Wangtao-001** [comment]
是的，同样得到同事的推荐，也特别喜欢LocalSend，但是公司办公电脑是Windows7 32位，提示无法安装，需要64位，请问怎样才能得到支持？

**Reply by NightFeather0615** [reply]
32位系統只能說是遙遙無期了，19年到現在都沒啥進展，主要還是市場太小了
可以看看 flutter/flutter#37777

**Reply by NightFeather0615** [reply]
不過 ARM 今年內倒是挺有機會
